packages/ui/src/theme/tokens/types/designToken.ts

```diff
@@ -5,6 +5,34 @@ import { Properties } from 'csstype';
  */
 export type DesignToken<ValueType = unknown> = { value: ValueType } | ValueType;
 
+/**
+ * Opening up our theme to accept design tokens we don't explicitly define.
+ * This helper type allows you to have a deeply nested object where the nodes are DesignTokens
+ */
+export type RecursiveDesignToken<
```

Comment:
Sort of annoying but can `RecursiveDesignToken` and `DesignTokenValue` be moved to the bottom of the file with the other type utils?